src/pkg/cli/tail.go

```diff
@@ -46,7 +46,10 @@ type P = client.Property // shorthand for tracking properties
 
 // ParseTimeOrDuration parses a time string or duration string (e.g. 1h30m) and returns a time.Time.
 // At a minimum, this function supports RFC3339Nano, Go durations, and our own TimestampFormat (local).
-func ParseTimeOrDuration(str string) (time.Time, error) {
+func ParseTimeOrDuration(str string, now time.Time) (time.Time, error) {
+	if str == "" {
+		return time.Time{}, nil
```

Comment:
Maybe we should be more explicit about the default time value, or add comment explaining the zero time means jan 1 year 0001.